backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -579,7 +579,7 @@ void OpenGLSdlGraphics3dManager::drawOverlay() {
 
 #if !defined(__amigaos4__) && !defined(__MORPHOS__)
 OpenGL::FrameBuffer *OpenGLSdlGraphics3dManager::createFramebuffer(uint width, uint height) {
-#if !defined(USE_GLES2)
+#if !USE_FORCED_GLES2
```

Comment:
I wonder what prevents us to support multisample support for gles2. it's capable in general

Reply:
I don't know, I adapted the changes without thinking about it.